osu.Game.Tournament/Screens/Editors/TeamEditorScreen.cs

```diff
@@ -298,7 +291,12 @@ private void load()
 
                     private void updatePanel() => Scheduler.AddOnce(() =>
                     {
-                        drawableContainer.Child = new UserGridPanel(user.ToAPIUser()) { Width = 300 };
+                        userPanelContainer.Child = new UserListPanel(user.ToAPIUser())
+                        {
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.CentreLeft,
+                            Scale = new Vector2(1.7f),
```

Comment:
Going to be a bit of a nitpick, but if we're adjusting tourney client visuals, then this scale looks a bit... comical? Like the scale makes the user panel wildly out of proportion with everything on the screen and just unbalances it.

Propose toning this down to something like

```suggestion
                            Scale = new Vector2(1.2f),
```

| before | after |
| :-: | :-: |
| ![1689884282](https://github.com/ppy/osu/assets/20418176/cef285da-6145-41aa-9cf0-0ab8de2357f7) | ![1689884297](https://github.com/ppy/osu/assets/20418176/e2b46af4-5d54-4dfe-b642-50ea6dcfbef8) |

Reply:
I made them big to align to the textbox at the left (which I'd make single-line instead but it's a bit of a PITA). Will see if I can fix the textbox..